pkg/constants/constants.go

```diff
@@ -121,6 +121,13 @@ const (
 	InferenceServiceDefaultLoggerPort = "8081"
 )
 
+// Labels to put on kservice
+const (
+	KServiceComponentLabel = "component"
+	KServiceModelLabel     = "model"
+	KServiceEndpointLabel  = "endpoint"
```

Comment:
nit: KService -> KFService. KService canonically refers to Knative.

Reply:
These labels are to go on the created Knative Service so that knative will then add them to the underlying resources. Have added a [new comment to explain](https://github.com/kubeflow/kfserving/pull/699#issuecomment-594379264) (finally giving the example I promised before)